src/platform/with_wgl/native_gl_context.rs

```diff
@@ -0,0 +1,144 @@
+use platform::NativeGLContextMethods;
+use std::ffi::CString;
+use std::os::raw::{c_void};
+use std::ptr;
+use std::sync::{Once, ONCE_INIT};
+
+use winapi;
+use user32;
+use kernel32;
+use super::wgl;
+use super::wgl_attributes::*;
+use gleam::gl;
+
+/// Wrapper to satisfy `Sync`.
+struct HMODULEWrapper(winapi::HMODULE);
+unsafe impl Sync for HMODULEWrapper {}
+
+lazy_static! {
+    static ref GL_LIB: Option<HMODULEWrapper>  = {
+        let p = unsafe{kernel32::LoadLibraryA(b"opengl32.dll\0".as_ptr() as *const _)};
+        if p.is_null() {
+            debug!("opengl32.dll not found!");
+            None
+        }
+        else {
+            debug!("opengl32.dll LOADED!");
+            Some(HMODULEWrapper(p))
+        }
+    };
+}
+
+static LOAD_GL: Once = ONCE_INIT;
+pub fn load_gl() {
```

Comment:
I see...

In general I'd prefer to do the GL context dance on `get_proc_address`, to align with how the rest of back-ends behave. I think this shouldn't be a problem, and even though it certainly feels hacky, the other options do seem hacky too..